web/js/panel.js

```diff
@@ -2860,7 +2860,7 @@ function $drawLayoutShape($widget) {
                 }
                 $gCtx.quadraticCurveTo(p[0], p[1], midR[0], midR[1]);
             } else {
-                jmri.log("ERROR: unexpected LayoutShape point type '" + lspt + "' for " + $widget.ide);
+                console.error("ERROR: unexpected LayoutShape point type '" + lspt + "' for " + $widget.ide);
```

Comment:
Note: You *cannot* rely on the console existing--there is no reason not to expect `console.error` to crash out the JavaScript completely on user's devices, so please revert this, or come up with a reliable alternative.

There is *zero* benefit to writing code to making unit testing easy if if crashes out entirely for users.